### ISSUE TITLE
Bug: dnsname not working in rhel 8.3

### ISSUE THREAD

**Issue by daniel-wtd** [containers/podman#8365]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

I am not able to use the dnsname plugin in rhel 8.3.

**Steps to reproduce the issue:**

1. `dnf install containernetworking-plugins dnsmasq -y`
2. `podman network create`
3. add the appropriate lines to /etc/cni/net.d/cni-podman1.conflist
    ```
    ...
    },
    {
      "type": "dnsname",
       "domainName": "dns.podman"
    }
    ```
4. `podman run -dt --network cni-podman1 httpd`

**Describe the results you received:**

Starting the container is not possible, but it is created.
The following error is present:

```
ERRO[0002] CNI network "cni-podman1" not found
Error: error configuring network namespace for container <id>: CNI network "cni-podman1" not found
```

**Describe the results you expected:**

container is started and dnsname is working.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
podman version 2.0.5
```

**Output of `podman info --debug`:**

```
(paste your output here)
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-2.0.5-5.module+el8.3.0+8221+97165c3f.x86_64
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide?**

No

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by Luap99** [comment]
If you need to add the dnsname plugin manually after `podman network create` than the dnsname plugin is not detected by podman.

**Comment by daniel-wtd** [comment]
Let me see, if downstream can shed some light to it. It SHOULD be supported since 8.2, where it was broken and should be fixed with 8.3.

I will update the issue.

**Comment by rhatdan** [comment]
This should be a bugzilla not an issue.  Issues should only reflect on the upstream.
